config/manifests/bases/freeipa-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+  name: freeipa-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: IDM is the Schema for the idms API
+      displayName: IDM
+      kind: IDM
+      name: idms.idmocp.redhat.com
+      version: v1alpha1
+  description: Deploy Freeipa in OpenShift
```

Comment:
Description to display in OLM installed operators and in OLM catalog from the OpenShift console.